ramp-database/rampdb/model/base.py

```diff
@@ -0,0 +1,155 @@
+import os
+import sys
+from math import ceil
+
+from sqlalchemy import orm
+from sqlalchemy.ext.declarative import declarative_base
+
+__all__ = [
+    'Model',
+    'ModelBase',
+    'BaseQuery',
```

Comment:
Do those BaseQuery and ModelBase need to be exposed in `rampdb.model` ? They are more internal?